DNA_RNA

```diff
@@ -0,0 +1,40 @@
+class DNA:
+    def __init__(self, sequence):
+        self.sequence = sequence
+        self.sequence_length = len(sequence)
+
+    def gc(self):
+        gc_counter = 0
+        for i in range(self.sequence_length):
+            if (self.sequence[i] == 'G') or (self.sequence[i] == 'C'):
+                 gc_counter += 1
+        gc_content = round(float((gc_counter / self.sequence_length) * 100), 1)
+        return gc_content
+
+    def reverse_complement(self):
+        dna_reverse_complement_dict = {'A': 'T', 'C': 'G', 'G': 'C', 'T': 'A'}
+        return ''.join([dna_reverse_complement_dict[base] for base in sequence[::-1]])
+
+    def transcribe(self):
+        dna_transcribe_dict = {'A': 'U', 'C': 'G', 'G': 'C', 'U': 'A'}
+        for i in self.sequence:
+            dna_template = self.sequence.replace('T', 'U')
+        return ''.join([dna_transcribe_dict[base] for base in dna_template])
+
+class RNA:
+    def __init__(self, sequence):
+        self.sequence = sequence
+        self.sequence_length = len(sequence)
+
```

Comment:
same problems with initializing by non-RNA characters and string representation of the class objects. See comments above.

Reply:
Added filter

Reply:
add lowercase (augc) letters to appropriate symbols, or you can only initialize using capitalized strings. Same for the Dna.

Reply:
Done